backend/infrahub/core/account.py

```diff
@@ -244,7 +257,7 @@ async def query_init(self, db: InfrahubDatabase, **kwargs: Any) -> None:
             ORDER BY r2.branch_level DESC, r2.from DESC, r1.branch_level DESC, r1.from DESC
             LIMIT 1
         }
-        WITH object_permission, object_permission_namespace, is_active AS opn_is_active
+        WITH object_permission, object_permission_description, object_permission_namespace, is_active AS opn_is_active
```

Comment:
I might be wrong here, but do we really need to care about the description in this context? To me the description was something that we'd see in the list view when looking at different permissions. I don't know if we need to fetch it like this?